mediachain/indexer/mc_ingest.py

```diff
@@ -340,7 +340,7 @@ def tail_blockchain(via_cli = False):
 
     cur = SimpleClient()
     
-    for art in cur.read_artefacts():
+    for art in cur.get_artefacts():
```

Comment:
Wouldn't have caught it, this code was intentionally supposed to be broken non-functional pseudo code, to show one possible draft of this "simple API" idea :) Maybe shouldn't have named the function `test*`.

Yeah we definitely shouldn't fix up too much in this file, or we'll forget what the bigger aim was here - figuring out the medium / long term direction of this API.

So when should we all have the short meeting, to discuss how to move forward with some of the ideas here, or otherwise rule them out?